go.mod

```diff
@@ -27,73 +27,64 @@ replace (
-	k8s.io/legacy-cloud-providers => github.com/rancher/kubernetes/staging/src/k8s.io/legacy-cloud-providers v1.18.6-k3s1
-	k8s.io/metrics => github.com/rancher/kubernetes/staging/src/k8s.io/metrics v1.18.6-k3s1
-	k8s.io/node-api => github.com/rancher/kubernetes/staging/src/k8s.io/node-api v1.18.6-k3s1
-	k8s.io/sample-apiserver => github.com/rancher/kubernetes/staging/src/k8s.io/sample-apiserver v1.18.6-k3s1
-	k8s.io/sample-cli-plugin => github.com/rancher/kubernetes/staging/src/k8s.io/sample-cli-plugin v1.18.6-k3s1
-	k8s.io/sample-controller => github.com/rancher/kubernetes/staging/src/k8s.io/sample-controller v1.18.6-k3s1
+	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc91.0.20200707015106-819fcc687efb
+	github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.0.3-0.20200520003142-237cc4f519e2
+	gopkg.in/square/go-jose.v2 => gopkg.in/square/go-jose.v2 v2.2.2
+	k8s.io/api => github.com/erikwilson/kubernetes/staging/src/k8s.io/api v1.19.0-rc3-k3s1
+	k8s.io/apiextensions-apiserver => github.com/erikwilson/kubernetes/staging/src/k8s.io/apiextensions-apiserver v1.19.0-rc3-k3s1
+	k8s.io/apimachinery => github.com/erikwilson/kubernetes/staging/src/k8s.io/apimachinery v1.19.0-rc3-k3s1
+	k8s.io/apiserver => github.com/erikwilson/kubernetes/staging/src/k8s.io/apiserver v1.19.0-rc3-k3s1
+	k8s.io/cli-runtime => github.com/erikwilson/kubernetes/staging/src/k8s.io/cli-runtime v1.19.0-rc3-k3s1
+	k8s.io/client-go => github.com/erikwilson/kubernetes/staging/src/k8s.io/client-go v1.19.0-rc3-k3s1
+	k8s.io/cloud-provider => github.com/erikwilson/kubernetes/staging/src/k8s.io/cloud-provider v1.19.0-rc3-k3s1
+	k8s.io/cluster-bootstrap => github.com/erikwilson/kubernetes/staging/src/k8s.io/cluster-bootstrap v1.19.0-rc3-k3s1
+	k8s.io/code-generator => github.com/erikwilson/kubernetes/staging/src/k8s.io/code-generator v1.19.0-rc3-k3s1
+	k8s.io/component-base => github.com/erikwilson/kubernetes/staging/src/k8s.io/component-base v1.19.0-rc3-k3s1
+	k8s.io/cri-api => github.com/erikwilson/kubernetes/staging/src/k8s.io/cri-api v1.19.0-rc3-k3s1
+	k8s.io/csi-translation-lib => github.com/erikwilson/kubernetes/staging/src/k8s.io/csi-translation-lib v1.19.0-rc3-k3s1
+	k8s.io/kube-aggregator => github.com/erikwilson/kubernetes/staging/src/k8s.io/kube-aggregator v1.19.0-rc3-k3s1
+	k8s.io/kube-controller-manager => github.com/erikwilson/kubernetes/staging/src/k8s.io/kube-controller-manager v1.19.0-rc3-k3s1
+	k8s.io/kube-proxy => github.com/erikwilson/kubernetes/staging/src/k8s.io/kube-proxy v1.19.0-rc3-k3s1
+	k8s.io/kube-scheduler => github.com/erikwilson/kubernetes/staging/src/k8s.io/kube-scheduler v1.19.0-rc3-k3s1
+	k8s.io/kubectl => github.com/erikwilson/kubernetes/staging/src/k8s.io/kubectl v1.19.0-rc3-k3s1
+	k8s.io/kubelet => github.com/erikwilson/kubernetes/staging/src/k8s.io/kubelet v1.19.0-rc3-k3s1
+	k8s.io/kubernetes => github.com/erikwilson/kubernetes v1.19.0-rc3-k3s1
```

Comment:
Could you update my patches with https://github.com/rootless-containers/usernetes/tree/master/src/patches/kubernetes ?

Not a blocker for 1.19, but will be needed to support rootless cgroup2

Reply:
Absolutely, thanks for pointing this out! :)